doc/source/notebooks/dataset_gsm8k_sft.ipynb

```diff
@@ -49,7 +49,18 @@
     "## Initialization\n",
     "\n",
     "We first need to initialize fairseq2 -- `setup_fairseq2()`.\n",
-    "This will load the configuration and register the assets, which allows us to interact with pre-defined datasets and models."
+    "This will load the configuration and register the assets, which allows us to interact with pre-defined datasets and models.\n",
+    "\n",
+    "> Prerequisite: Follow the [HuggingFace Datasets Tutorial](https://huggingface.co/docs/hub/en/datasets-downloading) to download the [gsm8k data](https://huggingface.co/datasets/facebook/fairseq2-lm-gsm8k) (formatted with fairseq2 flavor) to your local path (_e.g._ `/datasets/facebook/fairseq2-lm-gsm8k/`).\n",
+    "\n",
+    "<details>\n",
+    "<summary>[1 example datapoint in the sft jsonl]</summary>\n",
+    "\n",
+    "```json\n",
+    "{\"src\": \"<|start_header_id|>user<|end_header_id|>\\n\\nBrittany got a 78 on her first test. After her second test, her average rose to an 81. What grade did she get on her second test?<|eot_id|><|start_header_id|>assistant<|end_header_id|>\\n\\n\", \"tgt\": \"First multiply her average grade by the number of tests she took to find the total number of points she scored: 81 points * 2 = <<81*2=162>>162 points\\nThen subtract the number of points she scored on her first exam to find how many points she scored on her second exam: 162 points - 78 points = <<162-78=84>>84 points\\n#### 84\"}\n",
```

Comment:
can you do a pretty print of a dict here to get different key at different lines